src/renderers/common/BindGroup.js

```diff
@@ -2,11 +2,12 @@ let _id = 0;
 
 class BindGroup {
 
-	constructor( name = '', bindings = [], index = 0 ) {
+	constructor( name = '', bindings = [], index = 0, cacheKey ) {
 
 		this.name = name;
 		this.bindings = bindings;
 		this.index = index;
+		this.cacheKey = cacheKey;
```

Comment:
`cacheKey` seems to suggest a string, I think we could have `bindingsReference` or something similar.

Reply:
done